### PR TITLE
Change content shipping

### PR DESCRIPTION
**Pull request by claudiobelen**
## Description
Enable Shipping OFF for MLA (White List).

### Issue involved
Fixes #<issue>

### Checklist:
Before sending your contribution, please specify the following: 

- [ ] This request modifies code snippets. 
- [ ] The contribution aligns with the information stated in the repository wiki.

- [ ] Contribution incorporates an article not included until now, which must be added to indexes as a new section. 
- [x] Contribution includes new features. 
- [ ] New features were communicated in changelog.
- [x] Requires translations.

### CODE REVIEW

File: guides/payments/web-payment-checkout/latest/shipments.es.md [hunk 1]
```diff
@@ -0,0 +1,517 @@
...
+---
+
+# Ofrece Mercado Envíos
+
+Integra Mercado Envíos para recibir el pago de tus productos y gestionar su envío al mismo tiempo. Solo tienes que agregar los datos necesarios en tu preferencia y configurar los datos de tu negocio.
+
+
+## Requisito previo
+
+### Activa Mercado Envíos
```
Comment (pamalv7) [inline guides/payments/web-payment-checkout/latest/shipments.es.md:15]:
```suggestion
### &nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;Activa Mercado Envíos
```

File: guides/payments/web-payment-checkout/latest/shipments.es.md [hunk 2]
```diff
@@ -0,0 +1,517 @@
...
+>
+> Las dimensiones que indiques deben coincidir con las del paquete para que el carrier no te rechace el envío. Si no lo rechaza y las dimensiones son incorrectas, descontaremos esa diferencia del monto total de la cuenta del vendedor.
+
+
+
+## Puedes añadir otros tipos de envío
+
+Por defecto, vas a tener configurado el envío a cargo del comprador. Si quieres, puedes ofrecer envío gratis y/o retiro en domicilio.
+
+### Envío gratis
```
Comment (pamalv7) [inline guides/payments/web-payment-checkout/latest/shipments.es.md:147]:
```suggestion
### &nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;Envío gratis
```

File: guides/payments/web-payment-checkout/latest/shipments.es.md [hunk 3]
```diff
@@ -0,0 +1,517 @@
...
+===
+https://api.mercadolibre.com/users/_sellerId_/shipping_options?free_method=_shippingMethodId_&item_price=_price_&zip_code=_zipCode_&dimensions=_alto_x_ancho_x_largo_,_peso_
+===
+
+curl --location --request GET 'http://api.mercadolibre.com/users/179504451/shipping_options?free_method=182&item_price=718&zip_code=74474322&dimensions=2x11x16,88'
+
+```
+]]]
+
+### Retiro en domicilio
```
Comment (pamalv7) [inline guides/payments/web-payment-checkout/latest/shipments.es.md:244]:
```suggestion
### &nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;Retiro en domicilio
```

File: guides/payments/web-payment-checkout/latest/shipments.es.md [hunk 4]
```diff
@@ -0,0 +1,517 @@
...
+
+MercadoPago.DataStructures.Preference.Shipment shipments = new MercadoPago.DataStructures.Preference.Shipment();
+//...
+shipments.FreeMethods = new List<int> { 73328, 504945 };
+//...
+preference.Shipments = shipments;
+```
+]]]
+
+#### Simulador de costos de envíos
```
Comment (pamalv7) [inline guides/payments/web-payment-checkout/latest/shipments.es.md:228]:
```suggestion
#### &nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;Simulador de costos de envíos
```

File: guides/payments/web-payment-checkout/latest/shipments.es.md [hunk 5]
```diff
@@ -0,0 +1,517 @@
...
+# Ofrece Mercado Envíos
+
+Integra Mercado Envíos para recibir el pago de tus productos y gestionar su envío al mismo tiempo. Solo tienes que agregar los datos necesarios en tu preferencia y configurar los datos de tu negocio.
+
+
+## Requisito previo
+
+### Activa Mercado Envíos
+
+Desde la cuenta del vendedor, <a href="https://www.mercadopago.com.ar/business#shipping" target="_blank">ingresa a la sección de Tu negocio > Configuración</a> y activa la opción de Mercado Envíos.
```
[the remark this answers is not in the document]
Comment (claudiobelen) [reply guides/payments/web-payment-checkout/latest/shipments.es.md]: se aplican los cambios solicitados.

File: guides/payments/web-payment-checkout/latest/shipments.es.md [hunk 6]
```diff
@@ -0,0 +1,517 @@
...
+//...
+shipments.FreeMethods = new List<int> { 73328, 504945 };
+//...
+preference.Shipments = shipments;
+```
+]]]
+
+#### Simulador de costos de envíos
+
+Puedes simular costos desde la calculadora de envíos. Para hacerlo, debes reemplazar los valores de sellerId, shippingMethod Id, price, zipCode, alto, ancho, largo,peso de tu paquete.
```
[the remark this answers is not in the document]
Comment (claudiobelen) [reply guides/payments/web-payment-checkout/latest/shipments.es.md]: se aplican los cambios solicitados.

File: guides/payments/web-payment-checkout/latest/shipments.pt.md [hunk 7]
```diff
@@ -0,0 +1,13 @@
...
+- mla
+indexable: false  
+---
+
+<!-- -->
+> WARNING
+>
+> Important
+>
+> Desculpe! No momento, esta página não está disponível em português.
```
[the remark this answers is not in the document]
Comment (claudiobelen) [reply guides/payments/web-payment-checkout/latest/shipments.pt.md:12]: se aplican los cambios solicitados.

File: guides/payments/web-payment-checkout/latest/shipments.es.md (hunk 2 above)
Comment (claudiobelen) [reply guides/payments/web-payment-checkout/latest/shipments.es.md]: se aplican los cambios solicitados.

File: guides/payments/web-payment-checkout/latest/shipments.es.md (hunk 4 above)
Comment (claudiobelen) [reply guides/payments/web-payment-checkout/latest/shipments.es.md]: se aplican los cambios solicitados.

File: guides/payments/web-payment-checkout/latest/shipments.es.md (hunk 3 above)
Comment (claudiobelen) [reply guides/payments/web-payment-checkout/latest/shipments.es.md]: se aplican los cambios solicitados.

File: guides/payments/web-payment-checkout/latest/shipments.es.md (hunk 1 above)
Comment (claudiobelen) [reply guides/payments/web-payment-checkout/latest/shipments.es.md]: se aplican los cambios solicitados.

File: guides/payments/web-payment-checkout/latest/shipments.en.md [hunk 8]
```diff
@@ -0,0 +1,13 @@
...
+- mla
+indexable: false
+---
+
+<!-- -->
+> WARNING
+>
+> Important
+>
+> We're sorry! This page is currently not available in English.
```
[the remark this answers is not in the document]
Comment (claudiobelen) [reply guides/payments/web-payment-checkout/latest/shipments.en.md:12]: se aplican los cambios solicitados.

**Pull request merged**